### ISSUE TITLE
DEPRECATION WARNING: Single arity template handlers are deprecated

### ISSUE THREAD

**Issue by Fudoshiki** [judofyr/temple#122]
DEPRECATION WARNING: Single arity template handlers are deprecated.  Template handlers must
now accept two parameters, the view object and the source for the view object.
```
Change:
  >> #<Class:0x000000010a6cd788>#call(template)
To:
  >> #<Class:0x000000010a6cd788>#call(template, source)
 (called from block in register_as at /Users/fudoshiki/.rvm/gems/ruby-2.6.1/gems/temple-0.8.0/lib/temple/templates/rails.rb:21)
```

**Comment by mcgregordan** [comment]
Fixed in #121 (if merged)

**Comment by Fudoshiki** [comment]
Fixed in #121

**Comment by judofyr** [comment]
v0.8.1 has now been released.
